### PR TITLE
10184 add module snippet_baidu_map

### PR DESCRIPTION
**Pull request by Rona111**

### CODE REVIEW

**Comment by duanyp1991** [comment]
LGTM

**Comment by elicoidal** [comment]
Propose to OCA
Eric CAUDAL --
_Eric Caudal_ /[Founder and CEO]/
Skype: elico.corp. Phone: + 86 186 2136 1670 (Cell), + 86 21 6211 
8017/27/37 (Office)
_Elico Shanghai_ (Hong Kong/Shenzhen/Singapore) 
http://www.elico-corp.com /_Odoo Gold Partner, best Odoo Partner 2014 
for APAC_/

**Comment by Rona111** [comment]
cc @elicoidal @seb-elico @duanyp1991 @siyuancn

File: snippet_baidu_map/views/snippet_baidu_map.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
+
+        <template id="inherited" name="google map API inherited" inherit_id="snippet_google_map.s_google_map_API" active="True">
+            <xpath expr="//body/script[@src='https://maps.googleapis.com/maps/api/js?v=3.exp&#38;libraries=places']" position="replace">
+            </xpath>
```
Comment (siyuancn) [inline snippet_baidu_map/views/snippet_baidu_map.xml:7]: Where is the code for baidu map?

**Comment by noahzaozao** [comment]
LGTM

**Pull request merged**